svc-bgs-api/src/config/settings.yml

```diff
@@ -0,0 +1,11 @@
+# Settings (like the RabbitMQ connection options or API keys)
+# Loaded by setup.rb and put into a global variable.
+
+bgs:
+  application: VRO
+  client_station_id: 281
+  client_username: VROSYSACCT
+  ssl_cert_file: /app/certs/tls_bip.crt
+  ssl_cert_key_file: /app/certs/tls.key
+  ssl_ca_cert_file: /app/certs/va_all.crt
```

Comment:
So for local testing, these `ssl` settings shouldn't be needed. Cool!

Reply:
Yep. I figured if we wanted to make settings/config more consistent and constant across environments, it's easiest to handle moving parts in code.